gasLimits.js

```diff
@@ -1,37 +1,55 @@
+const promisify = require("es6-promisify").promisify;
 /**
- * This is where we define the gas limit number for starting ganache, doing migrations and running
- * `DaoCreator.forgeOrg`.
+ * Defines some useful gas limit constants, provides some useful gasLimit-related utilities.
  *
  * Since this module needs to be usable in several different contexts, and cannot depend on doing a build
- * (because one of the contexts in which is is used is the script that itself does the builds), we're
+ * (because one of the contexts in which it is used is the script that itself does the builds), we're
  * putting this at the root level and writing it in straight low-common-denominator ES5 javascript.
  */
 const gasLimitsConfig =
-  {
-    /**
-     * The gas limit used by Arc to create a DAO
-     */
-    "gasLimit_arc": 5600000,
-    /**
-     * The amount of gas needed for each founder when creating a DAO
-     */
-    "gasLimit_perFounder": 50000,
-    /**
-     * How many founders are already accounted-for in the gaslimit from Arc
-     */
-    "foundersInGasLimitArc": 3,
-    /**
-     * Gas limit appropriate for all other transactions besides creating DAOs
-     */
-    "gasLimit_runtime": 4543760,
-  };
+{
+  /**
+   * The gas limit used by Arc to forge a DAO with foundersInGasLimitArc founders
+   */
+  "gasLimit_arc": 5600000,
```

Comment:
this value can and should be taken from arc
(e.g. test/constant)

Reply:
Arc is a _dev_ dependency of Arc.js, so Arc can't be expected to be present in an application context.  Thus we manually copy this value from Arc.

Reply:
though you are importing arc .

Reply:
We aren't literally "importing" Arc.  But we do have a dev _dependency_ on it so we can grab the contract json files at contract migration time.

Because it is a dev dependency, applications will not pull in the Arc package.  There is no need for them to do so.

The only time this value is used is when we compute the amount of gas need to forge an org.  I have been considering adding the ability for applications to customize this gas limit via the arc.js configuration options, but thus far I've heard no need for it.